web/packages/teleterm/src/mainProcess/contextMenus/tabContextMenu.ts

```diff
@@ -86,13 +185,17 @@ export async function openTabContextMenu(
   options: TabContextMenuOptions
 ): Promise<void> {
   const mainOptions: MainTabContextMenuOptions = {
-    documentKind: options.documentKind,
+    document: options.document,
   };
-  const eventType = await ipcRenderer.invoke(
+  const response = await ipcRenderer.invoke(
     TabContextMenuEventChannel,
     mainOptions
   );
-  switch (eventType) {
+  if (!response) {
+    return;
+  }
+
+  switch (response.event) {
```

Comment:
This is `any`, could we manually cast `response` to a type, add a type to the Promise on line 48, and add a default case with `satisfies never`?